### PR TITLE
Garnett: fix immersive colours

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Fixes some colours intended for tablet-sized immersive cards that have bled into mobile cards

## What is the value of this and can you measure success?

Beauty

### Before

picture 439

### After

picture 438

## Tested in CODE?

Nah

### CODE REVIEW

**Review by GHaberis — APPROVED** [review]
👍

File: static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss [hunk 1]
```diff
@@ -217,5 +208,19 @@
                 @include immersiveSublinks();
             }
         }
+
+        @include mq($from: tablet) {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss]: Shouldn't this also be wrapped inside a card type class selector? I was under the impression that immersive cards worked in mobile too, except when in list-type cards, but I may be wrong 😅

Comment (SiAdcock) [reply static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss]: You're sharp-eyed even at this ungodly hour! Thanks man, I'm on it

**Review by regiskuckaertz — APPROVED** [review]

File: static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss [hunk 2]
```diff
@@ -217,5 +208,38 @@
                 @include immersiveSublinks();
             }
         }
+
+        &:not(.fc-item--list-media-mobile):not(.fc-item--list-mobile) {
+            @include mq($from: mobile) {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss]: Probably should add an `$until` breakpoint here.

Comment (regiskuckaertz) [reply static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss]: I don't know how I do it, honest 😅

**Comment by NataliaLKB** [comment]
Reverting this change while we deploy a fix (to minimise complications)

**Pull request merged**